src/main/webapp/consentGroupDocuments/ConsentGroupDocuments.js

```diff
@@ -63,6 +65,35 @@ class ConsentGroupDocuments extends Component {
     });
   };
 
+  getAssociatedProjects = () => {
+    ConsentGroup.getConsentCollectionLinks(this.props.serverURL, this.props.projectKey).then(response => {
+      this.setState({ associatedProjects: response.data.collectionLinks })
+    }).catch(error => {
+      this.setState({serverError: true});
+      console.error(error);
+    });
+  };
+
+  handleUnlinkProject = (target) => () => {
+    ConsentGroup.unlinkProject(this.props.serverURL, this.props.projectKey, target).then(result => {
+      this.getAssociatedProjects()
+    }).catch(error => {
+      this.setState({serverError: true});
+      console.error(error);
+    });
+  };
+
+  redirectToProject = (projectKey) => {
+    let key = projectKey.split("-");
+    let projectType = '';
+    if (key.length === 3) {
+      projectType = key[1].toLowerCase();
+    } else {
+      projectType = key[0].toLowerCase();
```

Comment:
I do think that splitting the project key string is a brittle way to get the project type. In groovy, we have code that figures out what the controller is, but I don't think we have anything in javascript for this. It might be a little overkill, but if we had an API method that said, "get me the projectType for this projectKey", this would be a much more reliable method. 

Not a requirement for this PR though.